.github/workflows/build.yml

```
@@ -153,36 +229,181 @@ jobs:
+            # Try to install ARM32 development libraries (optional)
+            sudo apt-get install -y \
+              libssl-dev:armhf \
+              libopus-dev:armhf || {
+              echo "ARM32 dev libraries not available, will use vendored versions"
+            }
+
+            echo "ARM32 cross-compilation toolchain installed successfully"
+          fi
 
       - name: Install cross-compilation tools
         if: matrix.target == 'aarch64-unknown-linux-gnu'
```

actionlint (1.7.7)

property "target" is not defined in object type {config: {artifact_name: string; cross_tools: string; docker_platform: string; features: string; name: string; target: string}}

(expression)